docs/reference/ml/df-analytics/apis/start-trained-model-deployment.asciidoc

```diff
@@ -0,0 +1,54 @@
+[role="xpack"]
+[testenv="basic"]
+[[start-trained-model-deployment]]
+= Start trained model deployment API
+[subs="attributes"]
+++++
+<titleabbrev>Start trained model deployment</titleabbrev>
+++++
+
```

Comment:
Would it be worth adding a short description here?

Reply:
These are placeholders for now. The terminology is expected to change from _deployment_ to something else and the concept of a _deployment_ is being redefined in the backend code. I wouldn't invest too much time into this at the moment because much of it is in flux